src/object/powerup.cpp

```diff
@@ -110,6 +110,11 @@ PowerUp::initialize()
   SoundManager::current()->preload("sounds/fire-flower.wav");
   SoundManager::current()->preload("sounds/gulp.wav");
 
+  if(m_type == EGG) {
+    m_shadesprite = SpriteManager::current()->create("images/powerups/egg/egg.sprite");
+    m_shadesprite->set_action("shadow");
+  }
```

Comment:
Fix this by placing this code snippet at the end of the after_editor_set function.